### PR TITLE
chapter_11

### PR DESCRIPTION
**Pull request by FirdausIsa**
ready

### CODE REVIEW

**Comment by FirdausIsa** [comment]
ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/account_activations.scss:1]: please delete this file

File: app/helpers/account_activations_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (anlvq-1058) [inline app/helpers/account_activations_helper.rb:1]: please delete this file

File: app/views/user_mailer/account_activation.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1>Sample App</h1>
+
+<p><%= t(:hi)%> <%= @user.name %> <%= t(:comma)%></p>
```
Comment (anlvq-1058) [inline app/views/user_mailer/account_activation.html.erb:3]:
```suggestion
<p><%= t(:hi)%> <%= @user.name %> <%= t(:comma) %></p>
```

File: config/locales/ja.yml [hunk 4]
```diff
@@ -46,3 +46,12 @@ ja:
...
   n_authorized: "許可されていません"
+
+  info: "Please check your email to activate your account."
+  w_activate_acc: "Welcome to the Sample App! Click on the link below to activate your
+account: "
+  hi: "Hi,"
+  activated: "Account activated!"
+  invalid_link: "Invalid activation link"
+  not_activated: "Account not activated. Check you email for the activation link."
+
```
Comment (anlvq-1058) [inline config/locales/ja.yml:57]: please fix your endline by remove all space in last line

File: db/seeds.rb [hunk 5]
```diff
@@ -24,5 +26,7 @@
   User.create!(name: name,
                email: email,
                password: password,
-               password_confirmation: password)
+               password_confirmation: password,
+               activated: true,
+               activated_at: Time.zone.now)
 end
```
Comment (anlvq-1058) [inline db/seeds.rb:32]: fix your endline

File: test/controllers/account_activations_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/account_activations_controller_test.rb:1]: delete this file please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by FirdausIsa** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**